x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -90,12 +98,23 @@ const navigationTree: NavigationTreeDefinition = {
               children: [
                 {
                   link: 'apm:services',
+                  getIsActive: ({ pathNameSerialized, prepend }) => {
+                    return (
+                      pathNameSerialized.startsWith(prepend('/app/apm/services')) ||
+                      pathNameSerialized.startsWith(prepend('/app/apm/service-map')) ||
+                      pathNameSerialized.startsWith(prepend('/app/apm/service-groups')) ||
+                      pathNameSerialized.startsWith(prepend('/app/apm/mobile-services'))
+                    );
+                  },
                 },
                 {
                   link: 'apm:traces',
```

Comment:
Can users enable trace explorer in serverless?

Reply:
Good catch. I see this one is closed https://github.com/elastic/kibana/issues/152734 so it will be enabled by default. I will update